### DISCUSSION TITLE
Problem to link Prometheus with Grafana !

### DISCUSSION THREAD

**Discussion by NinoxExe** [charlie-haley/omada_exporter#91] [Q&A]
Hello, I have a problem to link Prometheus with Grafana, it gives me the following message "ReadObject: expect { or , or } or n, but found <, error found in #1 byte of ...|<html> | ..., bigger context ...|<html> <head> <title>omada_exporter</title> <|... - There was an error returned querying the Prometheus API."
This is my first time using Prometheus and Grafana and I feel a bit lost. Would it be possible to get some help please ?

**Comment by charlie-haley** [comment]
Hey, are you trying to read the exporter `/metrics` endpoint directly from Grafana?

if so, you will need to deploy Prometheus and configure it to scrape the metrics endpoint of the exporter
https://prometheus.io/docs/prometheus/latest/getting_started/#configure-prometheus-to-monitor-the-sample-targets
